agent/src/main/java/com/microsoft/applicationinsights/agent/internal/agent/redis/JedisClassDataProvider.java

```diff
@@ -46,13 +46,13 @@ public JedisClassDataProvider(Map<String, ClassInstrumentationData> classesToIns
     public void add() {
         try {
             ClassInstrumentationData data =
-                    new ClassInstrumentationData(JEDIS_CLASS_NAME, InstrumentedClassType.OTHER)
+                    new ClassInstrumentationData(JEDIS_CLASS_NAME, InstrumentedClassType.Redis)
```

Comment:
From [this Wiki entry](https://github.com/Microsoft/ApplicationInsights-Java/wiki/Configure-Dependecy-Type-in-the-Agent-Configuration) it looks like agent already supports types other than `Other` though the configuration (and later though some method in Notification Handler or in Default Method Visitor) using `Type` filed rather than `Kind` field.
Since we are using `DefaultMethodVisitor` here and are changing Notification Handler as well - should we consider using existing type handling mechanism as Wiki suggests?

Reply:
If I understand this comment correctly, then you're asking for the ability to override the type via config. For Redis, I'm in favor of having the type being set out-of-box. Doing this via AI-Agent.xml would mean yet another piece of config the customer needs to set. It makes sense for generic cases classes, but since we want to support Redis out-of-box with minimal config, I would say this is not needed.